openbas-front/src/admin/components/simulations/simulation/reports/ExerciseReportContent.tsx

```diff
@@ -129,10 +129,38 @@ const ExerciseReportContent: React.FC<Props> = ({ report, exerciseId, canWrite =
             initialInjectComments={report?.report_injects_comments}
             injects={injects}
             style={{ width: '100%', marginTop: 20 }}
-            onCommentSubmit={(value: ReportInjectComment) => updateReportInjectCommentForExercise(exerciseId, report.report_id, value)}
+            onCommentSubmit={(value) => updateReportInjectCommentForExercise(exerciseId, report.report_id, value)}
           />
         )
       }
+      {displayModule(ReportInformationType.GLOBAL_OBSERVATION)
+        && <div style={{ width: '100%', marginTop: 20 }}>
+          <Typography variant="h4" gutterBottom>
+            {t('Notes')}
+          </Typography>
+            {canWrite
+              ? <SimpleRichTextField
+                  value={globalObservationRef.current}
+                  onChange={(value: string) => {
+                    globalObservationRef.current = value;
+                  }}
+                  style={{ height: 200, width: '100%' }}
+                  onBlur={() => updateReportGlobalObservation(exerciseId, report.report_id, {
+                    report_informations: report.report_informations,
+                    report_global_observation: globalObservationRef.current,
+                    report_name: report.report_name,
+                  } as ReportInput)
+                }
+                /> : <Paper variant="outlined" sx={{ padding: '10px 15px 10px 15px' }}>
+                  {globalObservationRef.current
+                    ? <div dangerouslySetInnerHTML={{ __html: globalObservationRef.current }}/>
+                    : <div style={{ textTransform: 'none' }}>
```

Comment:
Do we need to sanitize dangerouslySetInnerHTML? 
We use it for the articles with <ExpandableMarkdown source={article.article_content ?? ''}/>. Maybe we could use it here as well?